index.md

```diff
@@ -0,0 +1,20 @@
+Header Test
+# <h1> Header 1
+# <h2> Header 2
+# <h6> Header 3
+
+Image Test - Text is above image in the edit file
+![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)
+--> Image Test - Text is below image in the edit file - no spaces - Why does it format like this?

```

Comment:
This formats strange when you don't use a space above the text. If space added (nothing in line 8, it looks like this would format ok in the preview)